morphic.js

```diff
@@ -2690,9 +2689,7 @@ Morph.prototype.drawOn = function (aCanvas, aRect) {
 
 Morph.prototype.fullDrawOn = function (aCanvas, aRect) {
     var rectangle;
-    if (!this.isVisible) {
-        return null;
-    }
+    if (!this.isVisible) {return; }
```

Comment:
The Snap<em>!</em> style has brackets around all signle line cases. (IMO, the spacing semantics Jens' chose are a bit unconventional, but hey, it's his code and it's readable. :grinning:)

As far as `null !== undefined`...yes, these are different, but this was an edge case that I actually found using the linter. `fullDrawOn()` shouldn't actually return anything, and later in the function, it (implicitly) returns undefined. Also, the draw mechanisms aren't really supposed to return a value, so using the default `undefined` makes the most sense...